### PR TITLE
circleci

### PR DESCRIPTION
**Pull request by nathanleiby**

### CODE REVIEW

**Comment by nathanleiby** [comment]
fyi @mohit this repo is failing for same reason as https://github.com/Clever/kayvee-go/pull/48 .. re: gopkg.in setup and how we convert it from Drone to CircleCI

**Comment by nathanleiby** [comment]
@mohit would you mind moving this one over too?

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+machine:
+  post:
+  - cd $HOME && git clone --depth 1 -v git@github.com:clever/ci-scripts.git && cd ci-scripts && git show --oneline -s
+  - $HOME/ci-scripts/circleci/golang-install 1.7
+  - mkdir -p $HOME/.go_workspace/src/gopkg.in/Clever/pathio.v3
```
Comment (nathanleiby) [inline circle.yml]: could this step happen within `golang-move-project`?

Comment (nathanleiby) [inline circle.yml]: Ah, I see, it need to exist for cloning into the build dir.

Comment (mohit) [inline circle.yml]: yup, it does not work in the checkout section.

**Pull request merged**